### DISCUSSION TITLE
source.properties Not Found

### DISCUSSION THREAD

**Discussion by monzeromer-hub** [slint-ui/slint#6037] [General] [answered]
Hi, all

I'm trying to run the android example but I'm getting this error:

```bash
Using package slint-demo in /home/monzeromer/Projects/slint-demo/Cargo.toml
thread 'main' panicked at /home/monzeromer/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ndk-build-0.10.0/src/ndk.rs:89:14:
Failed to read source.properties: Os { code: 2, kind: NotFound, message: "No such file or directory" }
note: run with RUST_BACKTRACE=1 environment variable to display a backtrace
```

I used this command:

```bash
cargo apk run --target aarch64-linux-android --lib
```

I've everything setup correctly, Android SDK, NDK and platform tools.

**Comment by ogoffart** [comment, marked as answer]
I'm not exactly sure what the problem is there. Googling for the error message gives https://github.com/bevyengine/bevy/issues/3458 and https://github.com/bevyengine/bevy/issues/8619 pointing out that ANDROID_NDK_ROOT might not be set correctly.
